src/main/java/io/stargate/sgv2/jsonapi/exception/JsonApiException.java

```diff
@@ -59,6 +56,12 @@ public CommandResult get() {
     }
   }
 
+  public CommandResult.Error getCommandResultError(String message) {
+    Map<String, Object> fields =
+        Map.of("errorCode", errorCode.name(), "exceptionClass", this.getClass().getSimpleName());
```

Comment:
the `JsonApiException` was not including the `exceptionClass` before so I added it, I guess this is also helpful information..